### DISCUSSION TITLE
weird error squiggle

### DISCUSSION THREAD

**Discussion by deepanshumishra** [microsoft/vscode-cpptools#6857] [General]
Today I configured vs code to compile and run C++,
I was working with vectors in my code:

`#include <bits/stdc++.h>
using namespace std;
int main()
{

    vector<int> vect {10, 20, 30};
    for (int i = 0; i < vect.size(); i++)
        cout << vect[i] << endl;

    return 0;
}`
 and I don't know why but 

error squiggle came up just below `{` before 10 in vector declaration line, and when I hover over that it shows:

`no instance of constructor "std::vector<_Tp, _Alloc>::vector [with _Tp=int, _Alloc=std::allocator<int>]" matches the argument list -- argument types are: (int, int, int)`

but my code ran successfully.

but after disabling C/C++ extension red line(error squiggle) goes away, and everything is fine.
is that the issue with extension?

I've attached the screenshot.

![Screenshot (403)](https://user-images.githubusercontent.com/49090565/105989823-a8b31880-60c7-11eb-9442-1d4870a8aa96.png)

**Comment by sean-mcmanus** [comment]
I don't repro the issue. What compilerPath and cppStandard are you using? What version of the C/C++ extension are you using?

**Reply by deepanshumishra** [reply]
thanks for reply but I don't know how after few hours the issue got resolved itself.
I just enabled and disabled the extension .
btw using v1.2.0-insiders2 of C/C++ extension.

**Comment by pragatiranjan0517** [comment]
![Screenshot (26)](https://user-images.githubusercontent.com/78580442/106990518-6a3ded80-679a-11eb-9cab-d0c06ba0c77e.png)
my program is not running even though its correct.

**Reply by deepanshumishra** [reply]
I think you need to rename the file by removing spaces in between.

**Reply by sean-mcmanus** [reply]
Yeah, or surround the paths in `""`
